checkmate_front/src/types/types.ts

```diff
@@ -0,0 +1,36 @@
+
+interface VerifyRequest {
+  id: number
+  input: string
+}
+
+interface SourceWebsite{
+  favicon_link: string
+  name: string
+}
+
```

Comment:
Why Not Merge SourceWebsite and Source ? (Just an idea)

Reply:
Changed it to "Website", it will be reused in Source and Conversation